tools/Scripts/CopyPluginsToUnity.ps1

```diff
@@ -1,3 +1,7 @@
+param(
+    [switch] $DebugCompositor

```

Comment:
>[switch] $DebugCompositor [](start = 4, length = 25)

Maybe this should just be a "BuildFlavor" and it can be either Release or Debug, and all of the components can use the same type? Otherwise, it seems like it will be weird if we want to add a "DebugOpenCV" flag later

Reply:
This is beyond the scope of this change, copying opencv requires meta files for opencv_*d.dll files which we don't currently have. If there ever becomes a need to debug in this manner, i will change things, but for now I am going to just enable copying debug compositor flags.